optuna/samplers/_cmaes.py

```diff
@@ -108,6 +109,20 @@ def objective(trial):
                 to set this flag :obj:`True` when the :class:`~optuna.pruners.HyperbandPruner` is
                 used. Please see `the benchmark result
                 <https://github.com/optuna/optuna/pull/1229>`_ for the details.
+
+        restart_strategy:
+            Strategy for restarting CMA-ES optimization when converges to a local minimum.
+            If given :obj:`None`, CMA-ES will not restart (default).
+            If given 'ipop', CMA-ES will restart with increasing population size.
+            Please see also ``inc_popsize`` parameter.
+
+            .. note::
+                Added in v2.1.0 as an experimental feature. The interface may change in newer
+                versions without prior notice.
```

Comment:
I marked this argument as an experimental. But it doesn't need to do because I'm honestly not planning to change the interface.

Reply:
I removed experimental label at a2403fcc.